frontend/apps/web/app/(mgmt)/[account]/transformers/page.tsx

```diff
@@ -47,11 +50,16 @@ function TransformersTable(): ReactElement {
   if (transformersIsLoading || userDefinedTransformersLoading) {
     return <SkeletonTable />;
   }
-  const systemTransformerColumns = getSystemTransformerColumns();
+  const systemTransformerColumns = getSystemTransformerColumns({
+    accountName: '',
```

Comment:
This needs to pass in the real value, which comes from the account obj above.
`accountName: account?.name ?? ''`